### PR TITLE
Bump the memory request/limit for ip-masq-daemon.

### PR DESCRIPTION
**Pull request by dnardo**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
issue #47865
**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

**Comment by dchen1107** [comment]
cc/ @davidopp

File: cluster/addons/ip-masq-agent/ip-masq-agent.yaml [hunk 1]
```diff
@@ -21,10 +21,10 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 8Mi
+            memory: 16Mi
           requests:
```
Comment (gyliu513) [inline cluster/addons/ip-masq-agent/ip-masq-agent.yaml]: One question here, instead of bump the request here, why not just remove the request?

Comment (gyliu513) [inline cluster/addons/ip-masq-agent/ip-masq-agent.yaml]: @dnardo @dchen1107 hope can get some comments from you for this question even this is merged, thanks.

**Pull request merged**